dcos_installer/test_backend.py

```diff
@@ -86,7 +86,7 @@ def test_version(monkeypatch):
     monkeypatch.setenv('BOOTSTRAP_VARIANT', 'some-variant')
     version_data = subprocess.check_output(['dcos_installer', '--version']).decode()
     assert json.loads(version_data) == {
-        'version': '1.13.0-alpha',
+        'version': '1.14-dev',
```

Comment:
By the way; why would we not call this `1.14.0-dev`?

Reply:
Hi @jgehrcke , we have kept the major working branches as `X.Y-dev`

* https://github.com/dcos/dcos/blob/1.12/gen/calc.py
* https://github.com/dcos/dcos/commit/eeac4bacaa966ca83c2e2fea9c9bca9b0c7e8c5f#diff-3cd4ddcecc89d0887dd9cb3d5fabe995) and 1.13 should be renamed back `1.13-dev` now.

Only the minor branches of the format `X.Y.Z` (like 1.12.4) will have the version as 1.12.4 or 1.14.0, and we have not anything like 1.13.0-dev. I like to consider 1.13, 1.12, 1.11 series are development branches, hence with just `-dev`, and 1.13.x, and 1.12.x, the three number branches as release branches with proper release versions.

During `alpha`, and `beta` we continue the development in the development branch, so there could be sometime when we still mark the `-dev` as `alpha`, `beta` or `.0`, but it should change back to `-dev` when the release done, so that can be keep treating /stating those as `-dev` branches. 

HTH.